### DISCUSSION TITLE
Python folding

### DISCUSSION THREAD

**Discussion by robinparmar** [atom/atom#22329] [Q&A]
Excuse a newbie question, but is folding configurable?

I notice immediately that I cannot fold multiline Python comments (those starting with triple quotes).

**Comment by icecream17** [comment]
No, it's not configurable by the user - folding (and other language grammar) is achieved with some default language packages,
in this case [language-python](https://github.com/atom/language-python).

Well technically you can fork the repo, and add comment folding support then do `apm install` `apm link`

edit: see https://github.com/atom/language-python/pull/333
